### PR TITLE
feature(#184): change node version

### LINKED ISSUE #184: [node/npm] Update node to version 16
## Short description of the feature
Update docker containers to `node:16-alpine`, update package-lock files to version 2, and add node version to use for development in README.

[X] Relates to frontend
[X] Relates to the backend
[ ] Requires modification of GraphQL
[ ] Requires modification of Database
[ ] Requires creating a new service

### PR DESCRIPTION
**Pull request by noahmay**
closes #184

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ This repository contains:
 
 ## Prerequisites
 - Docker
-- `.env` file
-- nodejs and npm (development only)
+- `.env` file in the root directory
+- `.env` file in the frontend directory
```
Comment (emilwihlander) [inline README.md:10]: .env.local, right?

File: frontend/codegen.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: "http://localhost:4000/graphql"
+schema:  ${GRAPHQL_ADDRESS}
```
Comment (emilwihlander) [inline frontend/codegen.yml:2]: NEXT_PUBLIC

**Review by emilwihlander — CHANGES_REQUESTED** [review]
Två små saker

File: README.md (hunk 1 above)
Comment (noahmay) [reply README.md:10]: Stämmer

**Review by emilwihlander — APPROVED** [review]

**Pull request merged**